<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt1</class>
 <widget class="QDialog" name="Qt1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lb_time">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pic">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>400</width>
     <height>240</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>178</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pb_prev">
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_next">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>83</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rb_manual">
      <property name="text">
       <string>Manual</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rb_auto">
      <property name="text">
       <string>Auto</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>178</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pb_del">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_open">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cam_open">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="history">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>History</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="filenamecheck">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Custom Filename </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="speedinput">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="rlabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Resistence: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="timeok">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeinput">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>230</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TimeInput:</string>
   </property>
  </widget>
  <widget class="QLabel" name="tlabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed(ms/pic):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_sum">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_num">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeinput_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>80</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="commandtext">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>120</y>
     <width>281</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="commandinput">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Command Input:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="commandok">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>80</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alert:</string>
   </property>
  </widget>
  <widget class="QLabel" name="alert">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>340</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
